qucs/qucs/mouseactions.cpp

```diff
@@ -319,6 +356,8 @@ void MouseActions::MMoveElement(Schematic *Doc, QMouseEvent *Event)
  */
 void MouseActions::MMoveWire2(Schematic *Doc, QMouseEvent *Event)
 {
+//ALYS - i saved old code inside #if 0 block
```

Comment:
Felix, "git revert" and commit without #if 0 ...could save my PR? :)
also i see another one small bug, not solved in my current code. 
so i'll do another one commit.